I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 159, in script
    $ agression + 2
  File "game/script.rpy", line 159, in <module>
    $ agression + 2
NameError: name 'agression' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 159, in script
    $ agression + 2
  File "C:\Users\marle\Downloads\renpy-8.2.0-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\marle\Downloads\renpy-8.2.0-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 159, in <module>
    $ agression + 2
NameError: name 'agression' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.2.0.24012702
Derleth Manor 1.0
Wed Feb 21 14:42:00 2024
